domain-ee/ee-ep-merge-app/src/python_src/util/metric_logger.py

```diff
@@ -5,7 +5,7 @@
 import datadog_api_client.v1.api.metrics_api as metrics_v1
 import datadog_api_client.v2.api.metrics_api as metrics_v2
 from config import ENV
-from datadog_api_client import ApiClient, Configuration
+from datadog_api_client import AsyncApiClient, Configuration
```

Comment:
Just to double-check: switching from ApiClient to AsyncApiClient just switches the underlying concurrency model right? It doesn't change the implementation to fire-and-forget? (Or are we already using fire-and-forget?)